lib/modules/platform/azure/index.spec.ts

```diff
@@ -820,9 +820,9 @@ describe('modules/platform/azure/index', () => {
           .fn(() => prUpdateResults.shift())
           .mockName('updatePullRequest');
 
-        azureHelper.getMergeMethod = jest
-          .fn()
-          .mockReturnValue(GitPullRequestMergeStrategy.Squash);
+        azureHelper.getMergeMethod.mockReturnValue(
```

Comment:
You can use `mockResolvedValueOnce` instead of having to create a new `Promise` here
```suggestion
    azureHelper.getMergeMethod.mockResolvedValueOnce(
```